content/en/docs/Configuration/no-istiod.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "No Istiod Access"
+description: "Kiali working with no access to Istiod"
+---
+
+## Introduction
+
+There are different scenarios where Kiali is required to work with no access to the Istio registry:
+
+* When using other Service Mesh solutions that doesn't expose Istiod
+* Different [deployment models](https://istio.io/latest/docs/ops/deployment/deployment-models/#multiple-clusters) where Istiod is not available locally and remotely 
+
+## Configuration
+
+Kiali needs to be set up when the Istio registry is not accesible. This is done with a configuration item, istio_api_enabled.  
+By default, istio_api_enabled is true. 
+
+```yaml
+# ...
+spec:
+external_services:
+  istio:
+    istio_api_enabled: false
+# ...
+```
+
+## How does it affect Kiali
+
+When Istio registry is not available, there are some expected changes: 
+
+* The control plane metrics won't be available.
+* The proxy status won't be available in the workloads details view.
+* The namespace list will be obtained directly from the Kubernetes API, because it won't be possible to use the Istio cache. This could affect slightly the performance.
```

Comment:
```suggestion
* The namespace list will be obtained directly from the Kubernetes API, because it won't be possible to use the Istio cache. This may slightly impact performance.
```